pygeoapi/templates/tilematrixsets/tilematrixset.html

```diff
@@ -1,18 +1,19 @@
-{% extends "_base.html" %}
-{% block title %}{{ super() }} {% trans %}TileMatrixSet{% endtrans %} {% endblock %}
-{% block crumbs %}{{ super() }}
-/ <a href="../TileMatrixSets/">{% trans %}TileMatrixSets{% endtrans %}</a>
-/ <a href="../TileMatrixSets/{{ data['id'] }}?f=html">{{ data['id'] }}</a>
-{% endblock %}
-{% block body %}
+{% extends "_base.html" %} {% block title %}{{ super() }} {% trans
```

Comment:
Are changes to this file necessary?

Reply:
I didn't changed the files. But I think it was just reformatted by my editor.